.github/workflows/update.yml

```diff
@@ -29,12 +29,10 @@ jobs:
     - name: Fetch followers
       run: go run . -- followers > ../followers.txt
     - name: Commit results
-      run: |
-        set -e
-        git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
-        git config --global user.name "github-actions[bot]"
-
-        d="$(date --rfc-3339=s)"
-        git add --all
-        git commit --allow-empty -m "Periodic update: $d"
-        git push origin master
+      uses: EndBug/add-and-commit@v9
+      with:
+        committer_name: GitHub Actions
+        committer_email: 41898282+github-actions[bot]@users.noreply.github.com
+        fetch: false
+        message: update-${{ env.GITHUB_RUN_NUMBER }}
+        add: "*.txt --force"
```

Comment:
Why not --all

Reply:
I thought since only text files supposed to get update if something went south there could be more file. Switch to --all if makes more sense